core/src/main/java/com/linecorp/armeria/client/retry/AbstractRetryingClient.java

```diff
@@ -189,9 +189,13 @@ protected final boolean setResponseTimeout(ClientRequestContext ctx) {
         final long responseTimeoutMillis = ctx.attr(STATE).responseTimeoutMillis();
         if (responseTimeoutMillis < 0) {
             return false;
+        } else if (responseTimeoutMillis == 0) {
```

Comment:
`responseTimeoutMillis ` can never be 0 in this case. So I think we can refer this and add a comment describe it.

Reply:
I just checked the documentation. https://github.com/line/armeria/blob/02cc67a3bb4277cab0126178c931171815e7a4a6/core/src/main/java/com/linecorp/armeria/client/retry/AbstractRetryingClient.java#L307

As far as I understand `State.responseTimeoutMillis()` is able to return `0`. Do we need to update the Javadoc? That is wrong location? 😅

Reply:
Ah yes, I misread the code. Sorry for making you get confused. 😅

Reply:
Don't care. ❤️  I've learned a lot from your review.